core/src/main/java/nl/nn/adapterframework/core/IbisException.java

```diff
@@ -134,12 +134,18 @@ public String getMessage() {
 	}
 
 	public static String expandMessage(String msg, Throwable e) {
+		return expandMessage(msg, e, true);
+	}
+
+	public static String expandMessage(String msg, Throwable e, boolean prependExceptionType) {
 		String result=null;
 		List<String> msgChain = getMessages(e, msg);
 		Throwable t = e;
 		for(String message:msgChain) {
-			String exceptionType = t instanceof IbisException ? "" : "("+t.getClass().getSimpleName()+")";
-			message = Misc.concatStrings(exceptionType, " ", message);
+			if(prependExceptionType) {
```

Comment:
Het idee van deze prepend is/was dat sommige niet-ibis excepties in hun message alleen maar 'null' zeggen, en je het probleem uit de naam van de exceptie moet halen. Dat raak je kwijt als je prependExceptionType op false zet.
Is dat bewust?